loader.qml

```diff
@@ -136,47 +121,123 @@ ApplicationWindow {
-                                + "/usr/share/asteroid-launcher/watchfaces/" 
-                                + appRoot.nameOfWatchfaceToBeTested + ".qml?" 
-                                + Math.random();
-                    }
+                    ToolTip.text: qsTr("Generate preview images for the .thumbnails and watchfacepreview folders")
+                    onClicked: snapshotsTimer.running = true
                 }
             }
 
             RowLayout {
                 Layout.alignment: Qt.AlignCenter
 
-                CheckBox {
-                    id: twelveHourCheckBox
+                ColumnLayout {
+                    Layout.alignment: Qt.AlignCenter
+
+                    CheckBox {
+                        id: roundCheckBox
+
+                        font.pixelSize: 30
+                        text: "\u25EF"
+                        ToolTip.visible: hovered
+                        ToolTip.delay: 600
+                        ToolTip.text: qsTr("Show the watchface as it would look on a round watch")
+                        checked: true
+                    }
+
+                    CheckBox {
+                        id: ambientCheckBox
+
+                        font.pixelSize: 40
+                        text: "\u263D"
+                        ToolTip.visible: hovered
+                        ToolTip.delay: 600
+                        ToolTip.text: qsTr("Switch to AmbientMode on black background")
+                    }
+
+                    CheckBox {
+                        id: halfSize
+
+                        text: qsTr("320px")
+                        ToolTip.visible: hovered
+                        ToolTip.delay: 600
+                        ToolTip.text: qsTr("Scale down view to 320x320px from 640px")
+                    }
 
-                    text: qsTr("12h")
-                    ToolTip.delay: 600
-                    ToolTip.visible: hovered
-                    ToolTip.text: qsTr("Switch to 2x 12h day format with am/pm")
                 }
 
-                CheckBox {
-                    id: setStaticTimeCheckBox
+                ColumnLayout {
+                    Layout.alignment: Qt.AlignCenter
+
+                    CheckBox {
+                        id: twelveHourCheckBox
+
+                        text: qsTr("12h")
+                        ToolTip.delay: 600
+                        ToolTip.visible: hovered
+                        ToolTip.text: qsTr("Switch to 2x 12h day format with am/pm")
+                    }
+
+                    CheckBox {
+                        id: setStaticTimeCheckBox
+
+                        text: qsTr("Set Time")
+                        ToolTip.delay: 600
+                        ToolTip.visible: hovered
+                        ToolTip.text: qsTr("Set a custom time by draging the tumblers or use the mouse wheel above them")
+                        checked: false
+                    }
+
```

Comment:
I still see this empty line. Same comment as I did before, let's remove it for style consistency ;)

Reply:
Huh? What s going on there?
I was under the impression last commit for this file https://github.com/AsteroidOS/unofficial-watchfaces/pull/58/commits/be1d44c0035ef567e65a4e1ea07d3db709f50df5 looks like so
https://github.com/AsteroidOS/unofficial-watchfaces/blob/be1d44c0035ef567e65a4e1ea07d3db709f50df5/loader.qml#L238

Reply:
Mhh okay strange now I also don't see it anymore. Nevermind then!